formgroups/spatialtemporalcoverage.html

```diff
@@ -46,7 +46,7 @@
                             </div>
                         </div>
                         <!--Eingabefeld Description-->
-                        <div class="col-12 col-sm-12 col-md-12 col-lg-4 p-1">
+                        <div class="col-12 col-sm-12 col-md-12 col-lg-3 p-1">
```

Comment:
Datum kann man jetzt komplett sehen, das ist super! für meinen Geschmack muss die Zeit dafür nicht so groß sein (viel leere im Feld). Übersehen habe ich bisher, dass die Timezone auch nicht komplett lesbar ist. Könntest du das noch schick machen?

Reply:
> Datum kann man jetzt komplett sehen, das ist super! für meinen Geschmack muss die Zeit dafür nicht so groß sein (viel leere im Feld). Übersehen habe ich bisher, dass die Timezone auch nicht komplett lesbar ist. Könntest du das noch schick machen?

Also mit der aktuellen Struktur können wir die Größe von Times nur in Zusammenhang mit Dates verändern, da beide in der selben Spalte liegen und sich den Platz einer col-Klasse teilen. Ich hab jetzt aber trotzdem mal auf großen Bildschirmen Timezone 1/12 kleiner gemacht und dafür Description 1/12 mehr gegeben. Ist das für dich besser?

Reply:
> > Datum kann man jetzt komplett sehen, das ist super! für meinen Geschmack muss die Zeit dafür nicht so groß sein (viel leere im Feld). Übersehen habe ich bisher, dass die Timezone auch nicht komplett lesbar ist. Könntest du das noch schick machen?
> 
> Also mit der aktuellen Struktur können wir die Größe von Times nur in Zusammenhang mit Dates verändern, da beide in der selben Spalte liegen und sich den Platz einer col-Klasse teilen. Ich hab jetzt aber trotzdem mal auf großen Bildschirmen Timezone 1/12 kleiner gemacht und dafür Description 1/12 mehr gegeben. Ist das für dich besser?

Timezone kleiner heißt doch, der Inhalt des Feldes ist noch weniger lesbar. Ich wollte es besser lesbar haben, also andersrum :)

Reply:
> > > Datum kann man jetzt komplett sehen, das ist super! für meinen Geschmack muss die Zeit dafür nicht so groß sein (viel leere im Feld). Übersehen habe ich bisher, dass die Timezone auch nicht komplett lesbar ist. Könntest du das noch schick machen?
> > 
> > 
> > Also mit der aktuellen Struktur können wir die Größe von Times nur in Zusammenhang mit Dates verändern, da beide in der selben Spalte liegen und sich den Platz einer col-Klasse teilen. Ich hab jetzt aber trotzdem mal auf großen Bildschirmen Timezone 1/12 kleiner gemacht und dafür Description 1/12 mehr gegeben. Ist das für dich besser?
> 
> Timezone kleiner heißt doch, der Inhalt des Feldes ist noch weniger lesbar. Ich wollte es besser lesbar haben, also andersrum :)

Sollte jetzt aber nicht oberste Prio sein. Die Funktionalität von maps ist wichtiger. Ich kann so damit erstmal leben, kann ja auch noch ein issue aufschreiben, wenns mich doch stört.

Reply:
> > > > Datum kann man jetzt komplett sehen, das ist super! für meinen Geschmack muss die Zeit dafür nicht so groß sein (viel leere im Feld). Übersehen habe ich bisher, dass die Timezone auch nicht komplett lesbar ist. Könntest du das noch schick machen?
> > > 
> > > 
> > > Also mit der aktuellen Struktur können wir die Größe von Times nur in Zusammenhang mit Dates verändern, da beide in der selben Spalte liegen und sich den Platz einer col-Klasse teilen. Ich hab jetzt aber trotzdem mal auf großen Bildschirmen Timezone 1/12 kleiner gemacht und dafür Description 1/12 mehr gegeben. Ist das für dich besser?
> > 
> > 
> > Timezone kleiner heißt doch, der Inhalt des Feldes ist noch weniger lesbar. Ich wollte es besser lesbar haben, also andersrum :)
> 
> Sollte jetzt aber nicht oberste Prio sein. Die Funktionalität von maps ist wichtiger. Ich kann so damit erstmal leben, kann ja auch noch ein issue aufschreiben, wenns mich doch stört.

Okay, aber wo soll ich das dafür benötigte 1/12 der Seitenbreite dann wegnehmen?

Reply:
> > > > > Datum kann man jetzt komplett sehen, das ist super! für meinen Geschmack muss die Zeit dafür nicht so groß sein (viel leere im Feld). Übersehen habe ich bisher, dass die Timezone auch nicht komplett lesbar ist. Könntest du das noch schick machen?
> > > > 
> > > > 
> > > > Also mit der aktuellen Struktur können wir die Größe von Times nur in Zusammenhang mit Dates verändern, da beide in der selben Spalte liegen und sich den Platz einer col-Klasse teilen. Ich hab jetzt aber trotzdem mal auf großen Bildschirmen Timezone 1/12 kleiner gemacht und dafür Description 1/12 mehr gegeben. Ist das für dich besser?
> > > 
> > > 
> > > Timezone kleiner heißt doch, der Inhalt des Feldes ist noch weniger lesbar. Ich wollte es besser lesbar haben, also andersrum :)
> > 
> > 
> > Sollte jetzt aber nicht oberste Prio sein. Die Funktionalität von maps ist wichtiger. Ich kann so damit erstmal leben, kann ja auch noch ein issue aufschreiben, wenns mich doch stört.
> 
> Okay, aber wo soll ich das dafür benötigte 1/12 der Seitenbreite dann wegnehmen?

von description?

Reply:
> > Okay, aber wo soll ich das dafür benötigte 1/12 der Seitenbreite dann wegnehmen?
> 
> von description?

Done.